components/ResultsPage/RangeFilter.tsx

```diff
@@ -15,6 +16,14 @@ export default function RangeFilter({
 }: RangeFilterProps): ReactElement {
   const [controlledInput, setControlledInput] = useState(selected);
 
+  const courseIDs = [1000, 2000, 3000, 4000, 5000, 6000];
+  const courseLabels = courseIDs.map((id) => id.toString());
+
+  const marks = {};
+  courseIDs.forEach((id, index) => {
+    marks[id] = courseLabels[index];
+  });
```

Comment:
nit - if we're not using `courseLabels` later on, might as well just cut it out
```suggestion
  const marks = {};
  courseIDs.forEach((id) => {
    marks[id] = id.toString();
  });
```